Symfony/CS/Fixer/Contrib/PhpUnitStrictFixer.php

```diff
@@ -19,22 +19,18 @@
  */
 final class PhpUnitStrictFixer extends AbstractFixer
 {
-    private $phpUnitMethods = array(
+    private $configuration = array(
         'assertAttributeEquals' => 'assertAttributeSame',
         'assertAttributeNotEquals' => 'assertAttributeNotSame',
         'assertEquals' => 'assertSame',
         'assertNotEquals' => 'assertNotSame',
     );
 
-    public function configure(array $usingMethods = null)
+    public function configure(array $usingMethods)
     {
-        if (null === $usingMethods) {
-            return;
-        }
-
-        foreach (array_keys($this->phpUnitMethods) as $method) {
+        foreach (array_keys($this->configuration) as $method) {
             if (!in_array($method, $usingMethods, true)) {
-                unset($this->phpUnitMethods[$method]);
+                unset($this->configuration[$method]);
```

Comment:
I think we should go into `PhpUnitConstructFixer` way, so configuration with default one and allowing to override it - even partially. In `PhpUnitStrictFixer` configuration you may only change whole configuration. I think `PhpUnitStrictFixer` should be changed in way that `PhpUnitConstructFixer` works, but that is a BC break that can be done on master.